app/_src/gateway/kong-enterprise/aws-iam-auth-to-rds-database.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Use AWS IAM Authentication to connect to RDS database in Kong Gateway Enterprise
+badge: enterprise
+content_type: reference
+---
+
+Starting in version 3.3, {{site.base_gateway}} can be set up to use the AWS IAM Authentication to connect to the AWS RDS database(Postgres). This document describes the details about how to use this feature to secure your database configurations and database connections.
+
+
+## How does it work
+
+The AWS RDS database provides the ability to allow you to use the AWS Identity and Access Management(IAM) to authenticate and connect.
+With this feature enabled, you don't need to use a password when you connect to a DB instance. Instead, you use a temporary authentication token, which is a unique string of characters that generates on request. The authentication still goes through the standard database authentication, and you don't need to store user credentials in the database, because the authentication is managed externally using IAM. If you're a Kong Enterprise user in the AWS cloud environment, you can apply this feature with proper IAM configuration to your running cluster, so that you don't need to store any database user credentials on both Kong side and the RDS database side.
```

Comment:
Let's add a mention here to which password we refer and may be link to that configuration for reference so that users know that they can skip providing `pg_password` and DO NOT skip providing `kong_password` which is still required in bootstrapping for kong admin authentication.

Reply:
Done